pkgs/development/go-packages/generic/default.nix

```diff
@@ -81,7 +81,14 @@ let
       ++ (lib.optional (!dontRenameImports) govers) ++ nativeBuildInputs;
     buildInputs = buildInputs;
 
-    inherit (go) GOOS GOARCH;
+    inherit (go) GOOS GOARCH GO386 CGO_ENABLED;
+
+    GOHOSTARCH = go.GOHOSTARCH or null;
+    GOHOSTOS = go.GOHOSTOS or null;
+    CC_FOR_TARGET = go.CC_FOR_TARGET or null;
+    CXX_FOR_TARGET = go.CXX_FOR_TARGET or null;
```

Comment:
Does the build use `CC_FOR_TARGET` or `CXX_FOR_TARGET` in this case at all?

Reply:
Yeah I highly doubt you want the actual `CC_FOR_TARGET`, which is only used for building compilers.

Reply:
Ah, the names are totally overloaded and I am using these as a helper for setting `$CC` below like @Mic92 said.

Reply:
Specifying it here, makes it an environment variable in the build, which is not needed.